lib/secure_headers/middleware.rb

```diff
@@ -12,7 +10,8 @@ def call(env)
       status, headers, response = @app.call(env)
 
       config = SecureHeaders.config_for(req)
-      flag_cookies_as_secure!(headers) if config.secure_cookies
+      flag_cookies!(headers, config.cookies) if config.cookies
+      flag_cookies!(headers, secure: true) if config.secure_cookies
```

Comment:
Absolutely. I was going to ask you about that. I was thinking I'd add it on configuration